Dockerfile

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (C) 2011 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM centos:7
+
+RUN yum update -y && \
+  yum install -y centos-release-scl && \
+  yum install -y bzr \
+                 docker \
+                 gcc \
+                 hg \
+                 java-1.8.0-openjdk-devel \
+                 java-1.8.0-openjdk-devel.i686 \
+                 make \
+                 sclo-git212-git \
+                 svn \
+                 unzip \
+                 wget \
+                 which \
+                 zip \
+                 https://centos7.iuscommunity.org/ius-release.rpm && \
+  yum clean all
+
+ENV PATH=/opt/rh/sclo-git212/root/bin:$PATH
+
+RUN curl --retry 999 --retry-max-time 0  -sSL https://bintray.com/artifact/download/fabric8io/helm-ci/helm-v0.1.0%2B825f5ef-linux-amd64.zip > helm.zip && \
+  unzip helm.zip && \
+  rm -f helm.zip && \
+  mv helm /usr/bin/
+
+# Maven
+RUN curl -L http://mirrors.ukfast.co.uk/sites/ftp.apache.org/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz | tar -C /opt -xzv
+
+ENV M2_HOME /opt/apache-maven-3.3.9
+ENV maven.home $M2_HOME
+ENV M2 $M2_HOME/bin
+ENV PATH $M2:$PATH
```

Comment:
@rupalibehera 
Every command adds a new layer to the container. May be you can combine these commands like below?

```
ENV M2_HOME /opt/apache-maven-3.3.9 \
    maven.home $M2_HOME \
    M2 $M2_HOME/bin \
    PATH $M2:$PATH
```
There are other commands in this file which can be combined this way(avoiding to comment at all such instances)

Also, instead of setting an environment variable just before it is required, we can set all of them in the begining. Is there any specific reason why it has been done this way?

Reply:
@kishansagathiya , I think so it is more readable that way when we set env variable just before.